services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(movie_show_time,
+                         movie_id: int,
+                         cinema_hall_id: int):
+
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    MovieSession.objects.create(show_time=movie_show_time,
+                                cinema_hall=cinema_hall,
+                                movie=movie)

```

Comment:
You don't need to get movie and cinema_hall objects. You can only pass their ids to `cinema_hall_id` and `movie_id` arguments.